app/routes/random.tsx

```diff
@@ -0,0 +1,60 @@
+import {useEffect} from 'react'
+import type {LoaderFunction} from '@remix-run/cloudflare'
+import {useOutletContext, useLoaderData} from '@remix-run/react'
+import {reloadInBackgroundIfNeeded} from '~/server-utils/kv-cache'
+import {loadAllQuestions, QuestionState, QuestionStatus} from '~/server-utils/stampy'
+import {Header, Footer} from '~/components/layouts'
+import {Question} from '~/routes/questions/$question'
+import useQuestionStateInUrl from '~/hooks/useQuestionStateInUrl'
+
+function randomItem(arr: any[]) {
+  return arr[Math.floor(Math.random() * arr.length)]
+}
+
+export const loader = async ({request}: Parameters<LoaderFunction>[0]) => {
+  try {
+    const {data} = await loadAllQuestions(request)
+    return {
+      ...randomItem(data.filter((q) => q.status == QuestionStatus.LIVE_ON_SITE)),
+      questionState: QuestionState.OPEN,
+    }
+  } catch (e) {
+    console.error(e)
```

Comment:
what's the deal with the server side logging here, please? does it do anything different than letting the exception unhandled?

why not send the error to client a la https://github.com/StampyAI/stampy-ui/blob/master/app/routes/questions/%24question.tsx ?